frostsnapp/native/src/api.rs

```diff
@@ -823,16 +951,31 @@ impl Wallet {
 
 pub struct SignedTx {
     pub inner: RustOpaque<RTransaction>,
+    pub psbt: Option<RustOpaque<bitcoin::psbt::PartiallySignedTransaction>>,
 }
```

Comment:
One of the more significant changes: decided to add an Option of a PSBT to `UnsignedTx` and `SignedTx`, this was easiest, keeps APIs mostly the same.

Reply:
Separated Psbt into its own type in 0b00f4f29ff4adefa43eb4d78da4034d08dd9541